### DISCUSSION TITLE
how to  get list more than 1000 data at once with one pure api call?

### DISCUSSION THREAD

**Discussion by wishkj** [pocketbase/pocketbase#3646] [Q&A]
I'm building a system using the api function of pocketbase, can't I get list more than 1000 data at once with one pure api call?

Currently, it is divided into perpage, totalItems, totalPage, etc., so I would like to ask if there is a good way to import a list of data that may be more than 10,000 or more than 50000 in the future.

**Comment by ganigeorgiev** [comment]
This was asked several times before but at the moment the max `perPage` limit is fixed to 500 and can't be changed (_I may reconsider it in the future, but it can be easily abused by the users and needs further tests before enable customizing it; most likely it will be added somewhere together/after #1232_).
